### PR TITLE
Add replacement guides

### LINKED ISSUE #655: Add replacement guides section to home page
Screenshot 2022-08-26 at 16 07 05

### PR DESCRIPTION
**Pull request by dhmacs**
closes #655 

~~deploy_block 🛑 Blocked by #650~~

Adds replacements guides section:

Screenshot 2022-08-26 at 17 36 41

## QA

1. Visit Vercel preview
2. Go to [/Products/iphone-11-pro-screen](https://react-commerce-git-add-replacement-guides-ifixit.vercel.app/Products/iphone-11-pro-screen)
3. Verify that you can see the replacement guides

### CODE REVIEW

**Comment by jordycosta** [comment]
QA 🟢 
deploy_block 🟨 on https://github.com/iFixit/react-commerce/pull/650

**Replacement Guides** section is present 👍 
Followed the testing procedure shown above

section

**Comment by jordycosta** [comment]
un_deploy_block 🟩 

https://github.com/iFixit/react-commerce/pull/650 is merged ✔️

**Comment by jordycosta** [comment]
QA 🟢 Carry-over

https://github.com/iFixit/react-commerce/pull/656#issuecomment-1228708230
Also ensured that the guide links are working 👍

File: frontend/models/product.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import {
    MoneyV2,
    CurrencyCode,
 } from '@lib/shopify-storefront-sdk';
+import { z } from 'zod';
```
Comment (masonmcelvain) [inline frontend/models/product.ts:10]: Question: I see that [`zod recommends`](https://github.com/colinhacks/zod#requirements) using strict mode typescript, but I don't think we're doing that in `frontend/tsconfig.json`. Do we care? Seems like things are working, so maybe it doesn't matter.

File: frontend/templates/product/sections/ReplacementGuidesSection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                        minH={{
+                           base: 'auto',
+                           md: '150px',
+                        }}
+                        bg="white"
+                        overflow="hidden"
+                        borderColor="gray.300"
+                        borderWidth="1px"
+                        borderRadius="md"
+                        // align="center"
```
Comment (masonmcelvain) [inline frontend/templates/product/sections/ReplacementGuidesSection/index.tsx:81]: accidental comment

**Review by masonmcelvain — APPROVED** [review]
CR ✌🏻 Looks great! `zod` looks really useful, kudos for finding and using that.

**Pull request merged**